skimage/feature/fisher_vector.py

```diff
@@ -0,0 +1,223 @@
+"""
+fisher_vector.py - Implementation of the Fisher vector encoding algorithm
+
+This module contains the source code for Fisher vector computation. The
+computation is separated into two distinct steps, which are called separately
+by the user, namely:
+
+learn_gmm: Used to estimate the GMM for all vectors/descriptors computed for
+           all examples in the dataset (e.g. estimated using all the SIFT
+           vectors computed for all images in the dataset, or at least a subset
+           of this).
+
+compute_fisher_vector: Used to compute the Fisher vector representation for a
+                       single set of descriptors/vector (e.g. the SIFT
+                       descriptors for a single image in your dataset, or
+                       perhaps a test image).
+
+Reference: Perronnin, F. and Dance, C. Fisher kernels on Visual Vocabularies
+           for Image Categorization, IEEE Conference on Computer Vision and
+           Pattern Recognition, 2007
+
+Origin Author: Dan Oneata (Author of the original implementation for the Fisher
+vector computation using scikit-learn and NumPy. Subsequently ported to
+scikit-image (here) by other authors.)
+"""
+import numpy as np
+
+
+class FisherVectorException(Exception):
+    pass
+
+
+class DescriptorException(FisherVectorException):
+    pass
+
+
+def learn_gmm(descriptors, num_modes, **kwargs):
+    """Function to estimate a GMM given a set of descriptors and number of
+    modes (i.e. Gaussians). This function is essentially a wrapper around
+    the scikit-learn implementation of GMM, namely the
+    sklearn.mixture.GaussianMixture class.
```

Comment:
@scikit-image/core Any idea how to format `sklearn.mixture.GaussianMixture` so that it gets linked to in the docs?  I suspect we don't have this set up.

Reply:
I think just using
```
:func:`sklearn.mixture.GaussianMixture`
```
should work.

Cross-linking to docs from other projects requires the `sphinx.ext.intersphinx` extension to be enabled in `conf.py`. It looks like we already have it there and also have scikit-learn already present in the intersphinx mapping defined there:
https://github.com/scikit-image/scikit-image/blob/481ea168687f208da4cce9ef307ec635be43ba06/doc/source/conf.py#L387-L397

Reply:
Awesome, thanks Greg.

Reply:
Where should I add this line to? @stefanv @grlee77 Thanks

Reply:
I made a suggestion. If you accept that we can check if it is working as expected

Reply:
Done